tests/page-objects/default/enketo/common-enketo.wdio.page.js

```diff
@@ -0,0 +1,35 @@
+const currentSection = () => $('section[class*="current"]');
+
+const selectRadioButton = async (question, label) => {
+  const radioButton = await currentSection()
+    .$(`legend*=${question}`)
+    .parentElement()
+    .$(`label*=${label}`);
+  await radioButton.waitForClickable();
+  await radioButton.click();
+};
+
+const selectCheckBox = async (text) => {
+  let checkbox = await currentSection();
+  if (await checkbox.$('fieldset.or-branch:not(.disabled)').isExisting()){
+    checkbox = await checkbox.$('fieldset.or-branch:not(.disabled)');
+  }
+  checkbox = await checkbox.$(`label*=${text}`);
```

Comment:
I had a little trouble following this... I think it would be clearer if each had their own constant, eg:

```suggestion
  let section = await currentSection();
  if (await checkbox.$('fieldset.or-branch:not(.disabled)').isExisting()){
    section = await checkbox.$('fieldset.or-branch:not(.disabled)');
  }
  const checkbox = await section.$(`label*=${text}`);
```

I'm not quite sure why the `'fieldset.or-branch:not(.disabled)'` bit is needed, so that might need a comment, or a new function with a useful name, eg:

```js
const enabledFieldset = (section) => section.$('fieldset.or-branch:not(.disabled)');

const doSomethingWithUsefulName = async (section) => {
  const fieldset = enabledFieldset(section);
  if (await fieldset.isExisting()) {
    return fieldset;
  }
  return section;
};
```

Reply:
Thank you Gareth, for your early review, you are right, I am going to exclude it in a separate function.
It is in there because, for the pregnancy form, there is a `pre-init` fieldset that contains also some of the Risk Factors, so if I search for `label*=Heart condition` it will find two, and the first one is the one related to the `pre-init` that is also `disabled` so the test will not be able to click on the checkbox

![image](https://github.com/medic/cht-core/assets/94494491/8bf00ea2-b672-4784-b440-47979920fcfa)